synapse/storage/database.py

```diff
@@ -98,6 +98,8 @@
     "event_push_summary": "event_push_summary_unique_index2",
     "receipts_linearized": "receipts_linearized_unique_index",
     "receipts_graph": "receipts_graph_unique_index",
+    "profiles": "profiles_full_user_id_key_idx",
+    "user_filters": "full_users_filters_unique_idx",
```

Comment:
FTR, this is included in this PR because it is the one that begin using `full_user_id` for the upserts.